spec/lib/regexp_preview/single_line_spec.rb

```diff
@@ -0,0 +1,183 @@
+require 'spec_helper'
+
+describe RegexpPreview::SingleLine do
+  describe ".initialize" do
+    subject { RegexpPreview::SingleLine.new("log_file.log", format, params) }
+
+    describe "format" do
+      let :params do
+        {
+          regexp: "(?<regexp>regexp)",
```

Comment:
The word `regexp` is 3 way different meaning as 1) Hash key name, 2) named capture in Regexp, and 3) will match pattern.
That is a bit of confusing, rename them possible?